starter-core/src/main/java/io/micronaut/starter/feature/dekorate/DekorateTekton.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2020 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.starter.feature.dekorate;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import edu.umd.cs.findbugs.annotations.Nullable;
+import io.micronaut.starter.application.ApplicationType;
+import io.micronaut.starter.application.generator.GeneratorContext;
+import io.micronaut.starter.feature.Category;
+import io.micronaut.starter.feature.Feature;
+
+import javax.inject.Singleton;
+
+/**
+ * Adds Dekorate Tekton support.
+ *
+ * @author Pavol Gressa
+ * @since 2.0.1
+ */
+@Singleton
+public class DekorateTekton implements Feature {
+
+    @NonNull
+    @Override
+    public String getName() {
+        return "dekorate-tekton";
+    }
+
+    @Override
+    public String getTitle() {
+        return "Dekorate Tekton Support";
+    }
+
+    @Override
+    public String getDescription() {
+        return "Generates Tekton CI/CD kubernetes manifests using Dekorate Tekton Support.";
+    }
+
+    @Override
+    public String getCategory() {
+        return Category.CLOUD;
+    }
+
+    @Override
+    public void apply(GeneratorContext generatorContext) {
+    }
```

Comment:
No need to override these two methods, please remove them.

Reply:
sure